firefoxThemes/popup/popup.js

```diff
@@ -16,58 +18,21 @@ const backgroundTypes = {
 };
 
 /*Set color of popup menu based on theme*/
-function setCss(chosenTheme) {
+function setCSS(chosenTheme) {
   if (!chosenTheme) return
-  const {colors} = chosenTheme.definition
-  const styles = `
-.popup-header {
-  background-color: ${colors.headerColor};
-  color: ${colors.infoForeground};
-}
-
-
-* {
- color: ${colors.lineNumberColor};
-}
-
-.slider {
-  background-color: ${colors.selectionInactive};
-}
-
-.slider:before {
-  background-color: ${colors.accentColor};
-}
-
-select {
-  background-color: ${colors.buttonColor};
-  color: ${colors.buttonFont};
-}
-
-option {
-  color: ${colors.buttonFont};
-}
-
-.popup-body {
-  background-color: ${colors.baseBackground};
-}
+  const {colors, information} = chosenTheme.definition;
+  root.style.setProperty('--switch-shadow-color',information.dark ? 'white' : 'black');
```

Comment:
FYI, I actually did the icon theming in #64 . Will wait until this is in before I merge the other pull request in.

Reply:
The design looks nice! [Image reference.](https://user-images.githubusercontent.com/15972415/132136186-96a16e2b-2d9e-4219-a578-1dcee320d27d.png) Will the leftmost heart be used as an affordance for something?

Reply:
> Image reference. Will the leftmost heart be used as an affordance for something?

I thought about it, but since there is already a `home` link provided , I did not see the  point of that taking the user home too. Unless people are used to clicking on icons to go home. (was assuming you were speaking to the image reference leftmost heart)

Reply:
My apologizes. I was referencing the heart ![located here.](https://user-images.githubusercontent.com/23222943/132390768-b1442139-3137-418f-a638-d8037ad1b602.png)

Reply:
Ah, so that is the `Heart Fairy` that follows your mouse around on "The Doki Theme Home" website.

https://doki-theme.unthrottled.io/?themeId=5de1c901-2a12-411f-b75b-64e43aaf2c9d 

Wanted to reference that, because I themed the logo in that project too.

Reply:
oh, I see now! Interesting!

Reply:
I recommend adding a fairy after effect movement trail. Or include a group of fairies that follow.

If you are going to include the previously mentioned effects keep an eye out on performance.

Reply:
> I recommend adding a fairy after effect movement trail. Or include a group of fairies that follow.

:laughing: I hadn't thought of adding particle effects on moments. Thanks for the suggestion!